locksmith/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@unlock-protocol/locksmith",
-  "version": "1.1.0",
```

Comment:
Not sure we need that because this is not a package?

Reply:
Yeah, I know. It is to tied to openapi spec so we know what changed between.